bin/rose-suite-run

```diff
@@ -88,7 +88,9 @@
 #         Do not validate (suite engine configuration) in strict mode.
 #     --opt-conf-key=KEY, -O KEY
```

Comment:
Looks like this line needs updating too, should be:

--opt-conf-key=KEY, -O KEY, --opt-conf-key='(KEY)', -O '(KEY)'